src/views/services/form/ServiceFormPage.tsx

```
@@ -0,0 +1,71 @@
+  );
+
+  return (
+    <>
+      <PageSection>
+        <Title headingLevel="h2">
+          {isEditing
+            ? t('Edit {{label}}', { label: ServiceModel.label })
+            : t('Create {{label}}', { label: ServiceModel.label })}
+        </Title>
+        <Content component={ContentVariants.p}>
+          {t('Create a Service using the form. Switch to YAML view for full control, including fields not shown in the form.')}
```

ESLint

[error] Replace `'Create·a·Service·using·the·form.·Switch·to·YAML·view·for·full·control,·including·fields·not·shown·in·the·form.'` with `⏎············'Create·a·Service·using·the·form.·Switch·to·YAML·view·for·full·control,·including·fields·not·shown·in·the·form.',⏎··········`

(prettier/prettier)